### DISCUSSION TITLE
Unspecified namespace trpcNext

### DISCUSSION THREAD

**Discussion by jimfilippou** [trpc/trpc#2043] [Q&A] [answered]
Hello, first of all thank you for this amazing library, it has tremendously improved my workflow and the software actually ships faster.

I would like to point out that this file https://github.com/trpc/trpc/blob/main/www/docs/server/authorization.md has an unspecified namespace which is `trpcNext`. 

<img width="880" alt="image" src="https://user-images.githubusercontent.com/6877046/175813442-0983903d-f6ad-4963-94bb-2b5036cdc73d.png">

Where do I import this from? thank you in advance!

**Comment by sachinraja** [comment, marked as answer]
It's a missing import, `trpcNext` is this:
```ts
import * as trpcNext from '@trpc/server/adapters/next';
```

Also thanks for the kind words ♥️
